spec/facility_spec.rb

```diff
@@ -23,4 +35,101 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+
+  #Vehicle Registration
+
+  describe '#register vehicle' do
+    it 'can register_vehicle & count fees' do
+      expect(@cruz.registration_date).to eq(nil)
+      expect(@facility_1.registered_vehicles).to eq([])
+      expect(@facility_1.collected_fees).to eq(0)
+    
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(@cruz)      
+    
+      expect(@cruz.registration_date).to eq(Date.today)
+      expect(@cruz.plate_type).to eq(:regular)
+
+      expect(@facility_1.registered_vehicles).to eq([@cruz])
+      expect(@facility_1.collected_fees).to eq(100)
+
+      expect(@facility_1.register_vehicle(@camaro)).to eq([@cruz, @camaro])
+      
+      
+      expect(@camaro.registration_date).to eq(Date.today)
+      expect(@camaro.plate_type).to eq(:antique)      
+
+      expect(@facility_1.register_vehicle(@bolt)).to eq([@cruz, @camaro, @bolt])
+      expect(@bolt.registration_date).to eq(Date.today)
+      expect(@bolt.plate_type).to eq(:ev)
+      expect(@facility_1.registered_vehicles).to eq([@cruz, @camaro, @bolt])
+    
+      expect(@facility_1.collected_fees).to eq(325)
+
+
+      expect(@facility_2.registered_vehicles).to eq([])
+      expect(@facility_2.services).to eq([])
+      expect(@facility_2.register_vehicle(@bolt)).to eq(nil)
+      expect(@facility_2.registered_vehicles).to eq([])
+      expect(@facility_2.collected_fees).to eq(0)
+    end  
+  end
+  
+  describe 'Getting a Drivers Licence' do
+    it 'can input a drivers license' do
+
+  #Written Test
+    expect(@registrant_1.licensed_data).to eq({:written=>false, :license=>false, :renewed=>false})
+    expect(@registrant_1.permit).to eq(true)
+    expect(@facility_1.administer_written_test(@registrant_1)).to eq(false)
+    expect(@registrant_1.licensed_data).to eq({:written=>false, :license=>false, :renewed=>false})
+    expect(@facility_1.add_service('Written Test')).to eq(["Written Test"])
+    expect(@facility_1.administer_written_test(@registrant_1)).to eq(true)
+    expect(@registrant_1.licensed_data).to eq({:written=>true, :license=>false, :renewed=>false})
+    
+    
+    expect(@registrant_2.age).to eq(16)
+    expect(@registrant_2.permit?).to eq(false)
+    expect(@facility_1.administer_written_test(@registrant_2)).to eq(false)
+    @registrant_2.earn_permit
```

Comment:
Similarly, registrant tests should live in their own file. If tests were broken up and organized, you would have been closer to meeting testing expectations.